build.gradle

```diff
@@ -52,11 +52,24 @@ publishing {
     }
 }
 
+configurations {
+    // configuration that holds jars to include in the jar
+    extraLibs
```

Comment:
When using the library as a dependency in a new project the http builder is missing, i had to include it with the build outcomes.